static/js/logs.js

```diff
@@ -0,0 +1,24 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const renderLogs = (logs) => {
+    const logsNode = document.getElementById('logs');
+    const root = document.createElement('div')
+    logs.forEach( (el) => {
+        const pNode = document.createElement('p');
+        pNode.innerText = el
```

Comment:
nit: You could use a template literals string to do it in a single statement

Reply:
I don't think it's possible, because appendChild needs to have a Node argument.